From 45ac439a999ccc4797f96947657c1ea2ec6abaa8 Mon Sep 17 00:00:00 2001
From: "Loc Phuoc Pham.Lam" <loc.lam.px@renesas.com>
Date: Mon, 9 Sep 2019 13:11:03 +0700
Subject: [PATCH 261/326] arm64: dts: renesas: r8a774b1: Add SATA controller
 node

This patch adds SATA controller node for the R8A774B1 SoC.

Signed-off-by: Loc Phuoc Pham.Lam <loc.lam.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774b1.dtsi | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a774b1.dtsi b/arch/arm64/boot/dts/renesas/r8a774b1.dtsi
index cbe004b..e090c6c 100644
--- a/arch/arm64/boot/dts/renesas/r8a774b1.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a774b1.dtsi
@@ -1402,6 +1402,17 @@
 			};
 		};
 
+		sata: sata@ee300000 {
+			compatible = "renesas,sata-r8a774b1",
+						"renesas,rcar-gen3-sata";
+			reg = <0 0xee300000 0 0x200000>;
+			interrupts = <GIC_SPI 105 IRQ_TYPE_LEVEL_HIGH>;
+			clocks = <&cpg CPG_MOD 815>;
+			power-domains = <&sysc R8A774B1_PD_ALWAYS_ON>;
+			resets = <&cpg 815>;
+			status = "disabled";
+		};
+
 		pciec0: pcie@fe000000 {
 			compatible = "renesas,pcie-r8a774b1",
 						"renesas,pcie-rcar-gen3";
-- 
1.9.1

